sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubMessageSerializer.java

```diff
@@ -215,7 +215,7 @@ private EventData deserializeEventData(Message message) {
             body = bodyData.getValue().getArray();
         } else {
             logger.warning(String.format(Locale.US,
```

Comment:
@JonathanGiles @mclay Do you know how the string.format works with localisation? Is there something similar to CultureInfo.CurrentUICulture that C# has, which knows what the locale of the user is?

Reply:
`Locale.getDefault()`

Reply:
Ahh gotcha. Should remove the `String.format(Locale)` usage in favour of the overload without? from the docs, it looks like it uses Locale.getDefault(), by default with that overload.

Reply:
resolved